spring-boot-project/spring-boot-autoconfigure/build.gradle

```diff
@@ -58,6 +58,11 @@ dependencies {
 	}
 	optional("org.apache.httpcomponents.client5:httpclient5")
 	optional("org.apache.kafka:kafka-streams")
+	optional("org.apache.pulsar:pulsar-client-reactive-producer-cache-caffeine") {
+		// spring-pulsar-reactive includes the pulsar-client-reactive-adapter with
```

Comment:
The Reactive Pulsar client (`pulsar-client-reactive-adapter`) currently has a couple of prohibited deps that we filter out in Spring Pulsar (`spring-pulsar-reactive`). However, the `pulsar-client-reactive-producer-cache-caffeine` has a dep. on the `pulsar-client-reactive-adapter`. 

I am currently looking into splitting this out a bit in the client to avoid these annoying exclusions.